logprep/connector/http/input.py

```diff
@@ -432,8 +461,32 @@ class Config(Input.Config):
         be smaller than default value of 15.000 messages.
         """
 
-        collect_meta: str = field(validator=validators.instance_of(bool), default=True)
-        """Defines if metadata should be collected
+        copy_headers_to_logs: Set[str] = field(
```

Comment:
Should we use Set[str] over set[str]?
Not to challenge, but just for me to learn